scrapy/utils/log.py

```diff
@@ -235,15 +235,8 @@ def logformatter_adapter(logkws: dict) -> Tuple[int, str, dict]:
     if not {"level", "msg", "args"} <= set(logkws):
         warnings.warn("Missing keys in LogFormatter method", ScrapyDeprecationWarning)
```

Comment:
This also needs to be removed.

Reply:
Thanks for guiding. I've deleted these in the latest commit.